test/unit/memory-account.ts

```diff
@@ -43,23 +43,23 @@ describe('MemoryAccount', function () {
   })
 
   it('Init with secretKey as hex string', async () => {
-    const acc = MemoryAccount({ keypair: testAcc })
-    return acc.address().should.eventually.be.equal(testAcc.publicKey)
+    const acc: _AccountMemory = MemoryAccount({ keypair: testAcc })
```

Comment:
```suggestion
    const acc = MemoryAccount({ keypair: testAcc })
```
didn't we already defined `MemoryAccount` as `_AccountMemory`?